BedrockCore.cpp

```diff
@@ -268,6 +360,91 @@ BedrockCore::RESULT BedrockCore::processCommand(unique_ptr<BedrockCommand>& comm
     return needsCommit ? RESULT::NEEDS_COMMIT : RESULT::NO_COMMIT_REQUIRED;
 }
 
+BedrockCore::RESULT BedrockCore::postProcessCommand(unique_ptr<BedrockCommand>& command) {
+    AutoTimer timer(command, BedrockCommand::POSTPROCESS);
+    BedrockServer::ScopedStateSnapshot snapshot(_server);
+    command->lastPeekedOrProcessedInState = _server.getState();
+
+    // Convenience references to commonly used properties.
+    const SData& request = command->request;
+    SData& response = command->response;
+    STable& content = command->jsonContent;
+
+    // We catch any exception and handle in `_handleCommandException`.
+    try {
+        SDEBUG("postProcessing at '" << request.methodLine << "' with priority: " << command->priority);
+        uint64_t timeout = _getRemainingTime(command, false);
+        command->postProcessCount++;
+
+        _db.startTiming(timeout);
+
+        try {
+            if (!_db.beginTransaction(SQLite::TRANSACTION_TYPE::SHARED)) {
+                STHROW("501 Failed to begin shared postProcess transaction");
+            }
+
+            // Make sure no writes happen while in postProcess command
+            _db.setQueryOnly(true);
+
+            // postProcess.
+            command->reset(BedrockCommand::STAGE::POSTPROCESS);
+            bool completed = command->postProcess(_db);
```

Comment:
This is irrelevant here. postProcess is required to complete, there's nothing left to do after this.